### PR TITLE
only use Meta (Command) for xcv actions on macOS

### PR DESCRIPTION
**Pull request by kevinushey**
Closes https://github.com/rstudio/rstudio/issues/5696.

### CODE REVIEW

**Comment by gtritchie** [comment]
Was this a regression in 1.3?

**Comment by kevinushey** [comment]
That's correct. In particular, https://github.com/rstudio/rstudio/commit/0efd19ec67a9b87068bbbac89bec72a9e875e016#diff-0071cdcfd71c5ffde91ac336bae9017a is the offending commit.

**Comment by gtritchie** [comment]
Thanks, was just curious if this was related in some way to newer xterm.js or something else.

File: src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml [hunk 1]
```diff
@@ -575,10 +575,17 @@ well as menu structures (for main menu and popup menus).
...
-         <shortcut refid="pasteWithIndentDummy" value="Cmd+Shift+V" if="org.rstudio.studio.client.application.Desktop.isDesktop()"/>
+
+         <shortcut refid="cutDummy"             value="Ctrl+X"       if="org.rstudio.studio.client.application.Desktop.isDesktop() &amp;&amp; !org.rstudio.core.client.BrowseCap.isMacintosh()" />
+         <shortcut refid="copyDummy"            value="Ctrl+C"       if="org.rstudio.studio.client.application.Desktop.isDesktop() &amp;&amp; !org.rstudio.core.client.BrowseCap.isMacintosh()"/>
+         <shortcut refid="pasteDummy"           value="Ctrl+V"       if="org.rstudio.studio.client.application.Desktop.isDesktop() &amp;&amp; !org.rstudio.core.client.BrowseCap.isMacintosh()"/>
+         <shortcut refid="pasteWithIndentDummy" value="Ctrl+Shift+V" if="org.rstudio.studio.client.application.Desktop.isDesktop() &amp;&amp; !org.rstudio.core.client.BrowseCap.isMacintosh()"/>
+
+         <shortcut refid="cutDummy"             value="Meta+X"       if="org.rstudio.studio.client.application.Desktop.isDesktop() &amp;&amp; org.rstudio.core.client.BrowseCap.isMacintosh()" />
```
Comment (jmcphers) [inline src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml]: Nit: These tests would all read a lot cleaner as `isMacintoshDesktop()`

Comment (kevinushey) [reply src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml]: Right -- I chose this form mainly for symmetry with the above shortcuts.

**Pull request merged**